consensus/istanbul/backend/engine.go

```diff
@@ -390,6 +390,17 @@ func (sb *Backend) Prepare(chain consensus.ChainReader, header *types.Header) er
 	// wait for the timestamp of header, use this to adjust the block period
 	delay := time.Unix(header.Time.Int64(), 0).Sub(now())
 	time.Sleep(delay)
+
+	// modify the block header to include all the ParentSeals
+	parentCommitedSeals := sb.core.ParentSeals()
+	if parentCommitedSeals != nil {
```

Comment:
I do not think it should return an error, since for whatever reason maybe your connection dropped and you were not gossiped any ParentCommits. Maybe on error we could fetch the seals from our previous block and just use that (which is 2f+1, but better than 0)